### ISSUE TITLE
Quest: Razormaw

### ISSUE THREAD

**Issue by Sagrath** [dalaranwow/dalaran-wow#61]
http://wotlk.openwow.com/quest=9689
Razormaw dose not spawn after using the quest item.

**Comment by Uthil** [comment]
As for 4 Jan 2015 this quest is still **bugged / broken**. You can get the [Bundle of Dragonbones](http://wotlk.openwow.com/item=24221) to use on the fire but unfortunately it doesn't respond. Also another proof is that Razormaw isn't flying around that hill as he used to be.

**Comment by Portotada** [comment]
This reminds me of some bad memories of a bad server with many bugged quests that werent getting fixed, although they were reported.

**Comment by taeurus** [comment]
Razormaw is still not spawning for the quest and is not even visible at all in the sky. I have started, activated, dropped, restarted, reactivated, and redropped this quest several time now and I would like to finish it as it is my only quest left in this area.

**Comment by ironikx** [comment]
Razormaw quest is not bugged... After clicking the bones on the bonfire, you have to wait for 5-10 minutes and razormaw will spawn.

Regards

**Comment by Augur89** [comment]
Well, good to know that. Still I would consider it bugged, as no one should have to wait 5-10 minutes for a mob to spawn after you activate the required script...

**Comment by waefre1** [comment]
Yeah, it would still be bugged because IIRC Razormaw should be flying around above the island (he isn't) and then land within 1-3 minutes (he doesn't), right between the two stone monument things to the south of the plateau (he doesn't).

**Comment by n-ghost** [comment]
Waiting for a half an hour now, doesn't spawn for me. Should be looked into.

**Comment by necropola** [comment]
Seems like he is on a fixed timer completely unrlated to when you drop teh bones into the fire. I waited for around 55 minutes until 3 minutes before the hour. Then he finally spawned. BTW, he hits like a truck (possibly bugged). 5x250 within one second with his melee single target attack. So you have to kite him.

This is similar to the extremely high speed ranged attack of Matis the Cruel which forces you to let the NPC do the work almost entirely,

**Comment by joelmatheny** [comment]
Razormaw spawned just fine for me - dropped the bones, and he took a min or two, but slowly came down. While I was fighting him, he feared me with terrifying roar - which is fine, but it caused me to run right off the cliff, which normally did not happen. When I fell, it bugged, because I fell through the ground, and was suspended in the air falling. After a moment or two of that I died, and the release spirit button was greyed out - I waited the 6 minutes for spirit release, and was resurrected in the Sentinel Hill graveyard! The second time I fought him I was feared and I did not fall. The third time I was feared and fell through the ground without falling off the cliff, and I resurrected in the Blood Watch graveyard.
